tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs

```diff
@@ -6,6 +6,8 @@ namespace NHSD.BuyingCatalogue.Testing.Data.EntityBuilders
     public sealed class SolutionEpicEntityBuilder
     {
         private readonly SolutionEpicEntity _solutionEpicEntity;
+        private const int StatusId = 1;
```

Comment:
I was thinking here that the name of the constant could provide a bit more meaning. What does a status of 1 refer to?